### PR TITLE
Feature/connection management

### PR DESCRIPTION
**Pull request by Shubhangi-cs**

### CODE REVIEW

**Comment by rmstar** [comment]
Please fix e2e test failures

**Comment by Vipinofficial11** [comment]
> Please fix e2e test failures

The e2

> Please fix e2e test failures

The e2e tests are not updated yet based on the latest framework that's why there are failures.I Will raise a PR to update these tests its in queue.(Ariba -> Salesforce Marketing -> Salesforce)

**Comment by Vipinofficial11** [comment]
> > Please fix e2e test failures
> 
> The e2
> 
> > Please fix e2e test failures
> 
> The e2e tests are not updated yet based on the latest framework changes that's why there are failures.I Will raise a PR to update these tests its in queue.(Ariba -> Salesforce Marketing -> Salesforce)

File: src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java [hunk 1]
```diff
@@ -78,19 +79,17 @@ public class BaseSalesforceConfig extends ReferencePluginConfig {
   @Description("Endpoint to authenticate to")
   @Macro
   @Nullable
-  private String loginUrl;
+  public String loginUrl;
 
   private static final String DEFAULT_LOGIN_URL = "https://login.salesforce.com/services/oauth2/token";
```
Comment (rmstar) [inline src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java]: DEFAULT_LOGIN_URL is defined in 3 config classes. Can we dedupe?

File: src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java [hunk 2]
```diff
@@ -78,19 +79,17 @@ public class BaseSalesforceConfig extends ReferencePluginConfig {
   @Description("Endpoint to authenticate to")
   @Macro
   @Nullable
-  private String loginUrl;
+  public String loginUrl;
```
Comment (rmstar) [inline src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java]: nit: No need to make it public, just use the getter - getLoginUrl()

**Review by rmstar — APPROVED** [review]
Just a couple more minor comments. Please address them and squash commits

File: src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java (hunk 2 above)
Comment (Shubhangi-cs) [reply src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java]: done

File: src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java (hunk 1 above)
Comment (Shubhangi-cs) [reply src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java]: No, we can't.

**Pull request merged**